doc: document all available plug-ins (#1436)
openscd/open-scd: README.md

If you don't have your own webserver but still want your own version of OpenSCD hosted locally (e.g. on a system without an internet connection), you can [use a browser plugin that acts as a local webserver](https://github.com/openscd/open-scd/wiki/Install-OpenSCD#offline) (only for you) instead.

> ## Plugins

Reviewer: ```suggestion
## Plug-ins
```

> We gather the available plugins from the community in the [plugins](docs/plugins.md) file.  

Reviewer: ```suggestion
We gather the available plugins from the community in the [plug-ins](docs/plugins.md) file.  
```

> If you would like to list your plugin here, please open a pull request.

Reviewer: ```suggestion
If you would like to list your plug-in here, please open a pull request.
```

> If you would like to list your plugin here, please open a pull request.

Author: I'd say that the option to remove plugin is implicitly in this description.
Also, I think we can ask it in the PR if necessary.
